miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/SampleStockDto.java

```diff
@@ -0,0 +1,22 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+import org.codehaus.jackson.annotate.JsonTypeName;
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+
+import uk.ac.bbsrc.tgac.miso.core.data.SampleStock;
+
+@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+@JsonTypeName(value = SampleStock.CATEGORY_NAME)
+public class SampleStockDto extends SampleTissueDto {
```

Comment:
Concentration field missing. Should be moved from SampleAdditionalInfoDto to here